### PR TITLE
Adding realsense_ros_camera to documentation index for Kinetic and Indigo distro

### PR DESCRIPTION
**Pull request by icarpis**
I'd like `realsense_ros_camera` to be indexed and documented on ros.org.

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -10654,6 +10654,12 @@ repositories:
       url: https://github.com/intel-ros/realsense.git
       version: indigo-devel
     status: maintained
+  realsense_ros_camera:
+    doc:
+      type: git
+      url: https://github.com/intel-ros/realsense.git
```
Comment (tfoote) [inline indigo/distribution.yaml]: This looks to be a duplicate of: https://github.com/ros/rosdistro/pull/17045/files#diff-4bbb8e35ff5dc5e7f2d7b4d00374533fR10644 with a different branch. Do you want to change the documentation branch? If so then changing lin 10645 would be the right thing to do.

**Review by tfoote — CHANGES_REQUESTED** [review]
It looks like realsense_ros_camera is already submitted and indexed. Can you explain what you're looking for to be added?

**Comment by tfoote** [comment]
It looks like the driver has been renamed in the new branch?
Related to that in general our naming convention is to not include 'ros' in the ROS package name as it's redundant. http://www.ros.org/reps/rep-0144.html http://wiki.ros.org/ROS/Patterns/Conventions

**Comment by icarpis** [comment]
The [realsense_camera](http://wiki.ros.org/realsense_camera) and [realsense_ros_camera](http://wiki.ros.org/realsense_ros_camera) are independent packages. The first one works with [librealsense1](http://wiki.ros.org/librealsense) and the second one works with [librealsense2](http://wiki.ros.org/librealsense2). They are both located at the same [repository](https://github.com/intel-ros/realsense) but in different branches. The `realsense_camera` is located at the [indigo-devel](https://github.com/intel-ros/realsense/tree/indigo-devel) branch and the `realsense_ros_camera` is located at the [development](https://github.com/intel-ros/realsense/tree/development) branch.

**Comment by dirk-thomas** [comment]
> The first one works with librealsense1 and the second one works with librealsense2.

Maybe choose a different repository name then? The additional `ros` doesn't express the difference. Maybe `realsense2_camera`?

**Comment by icarpis** [comment]
Ok, I'll rename the repository name and apply the change to this PR.

**Comment by dirk-thomas** [comment]
Please see the Travis log for errors. The new entry needs to be inserted before `realsense_camera`.

**Review by tfoote — APPROVED** [review]

**Pull request merged**